npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -21,10 +21,12 @@ const (
 	doesNotExistErrorCode      int = 1 // Bad rule (does a matching rule exist in that chain?)
 	couldntLoadTargetErrorCode int = 2 // Couldn't load target `AZURE-NPM-EGRESS':No such file or directory
 
-	minLineNumberStringLength int    = 3
-	azureChainGrepPattern     string = "Chain AZURE-NPM"
-	minAzureChainStringLength int    = len(azureChainGrepPattern)
-	azureChainStartIndex      int    = 6
+	minLineNumberStringLength int = 3 // TODO transferred from iptm.go and not sure why this length is important, but will update the function its used in later anyways
```

Comment:
Nitpick: curious, is it idiomatic to put comment just next to code?
If not, if the comment is long, it would be nice to put above the line since I think it is more readable.

But we can update it since we need to resolve the question.

Reply:
just habit. I'll put comments on top now, but ya this variable will be deleted next PR anyways